File: 2356.js
// Node.js 12.18.3
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

for (let i = 0; i < lines.length; i+= 2) {
  const d = lines[i];
  const s = lines[i+1];

  // avoid runtime error
  if(d === undefined || s === undefined) {
    return;
  }

  if(d.includes(s)) {
    console.log('Resistente');
  } else {
    console.log('Nao resistente');
  }
}

File: 2356.py
# Python 3.4.3

def main():
  while True:
    try:
      d = input()
      s = input()

      if s in d:
        print('Resistente')
      else:
        print('Nao resistente')

    except EOFError:
      break

if __name__ == '__main__':
  main()
